src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -1,31 +1,51 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import path from 'ramda/src/path';
 import pathOr from 'ramda/src/pathOr';
+import styled from 'styled-components';
 import { Link } from 'react-router-dom';
-import { GhostGrid, GridItemConstrainedMedium } from '#lib/styledGrid';
+import { GhostGrid, GridItemConstrainedLarge } from '#lib/styledGrid';
+import { RequestContext } from '#contexts/RequestContext';
+
 import MetadataContainer from '../Metadata';
 import LinkedData from '../LinkedData';
 import headings from '../Headings';
 import timestamp from '../ArticleTimestamp';
 import text from '../Text';
 import image from '../Image';
 import Blocks from '../Blocks';
+import MediaPlayer from '../MediaPlayer';
 import ATIAnalytics from '../ATIAnalytics';
 import cpsAssetPagePropTypes from '../../models/propTypes/cpsAssetPage';
 
-const componentsToRender = {
-  headline: headings,
-  text,
-  image,
-  timestamp,
-};
-
 const CpsAssetPageMain = ({ pageData }) => {
+  const { platform } = useContext(RequestContext);
   const title = path(['promo', 'headlines', 'headline'], pageData);
   const summary = path(['promo', 'summary'], pageData);
   const metadata = path(['metadata'], pageData);
+  const assetUri = path(['locators', 'assetUri'], metadata);
   const blocks = pathOr([], ['content', 'model', 'blocks'], pageData);
 
+  const componentsToRender = {
+    headline: headings,
+    text,
+    image,
+    timestamp,
+    video: props => (
+      <MediaPlayer
+        {...props}
+        style={{ paddingBottom: 50 }}
+        embedOverrides={{
+          type: 'cps',
+          id: assetUri.substr(1),
+          showPlaceholder: platform === 'amp',
+          wrapper: styled(GridItemConstrainedLarge)`
```

Comment:
Rather than passing in an overridden wrapper, we could have a file that pulls in `MediaPlayer` and expands it to have the padding bottom.

Reply:
This is not just for the padding - we need to override articles' wrapper.  They're using 6 cols (`GridItemConstrainedMedium`) - our design uses 8 cols (`GridItemConstrainedLarge`)